### ISSUE TITLE
concurrency agent not working reliably with multiple clusters

### ISSUE THREAD

**Issue by alexkli** [apache/openwhisk-wskdebug#66]
With multiple clusters Openwhisk can create multiple containers for a concurrent action even with minimal traffic. Invocations in a cluster will create a new container in the same cluster. If invocations (actual invocation to debug vs. wskdebug polling) happen in different clusters, there will be 2 containers and they won't see each other, and wskdebug will fail to catch any invocations. This has been observed as a new behavior.

[Some internal discussion](https://cq-dev.slack.com/archives/C68UJPDM5/p1590554686290700).

**Comment by rabbah** [comment]
@alexkli the slack link is inaccessible to the slack community - can you summarize relevant parts?

**Comment by alexkli** [comment]
The summary is above :-) The link was a note for me to find it later if I need it.

**Comment by rabbah** [comment]
Thanks!

**Comment by shazron** [comment]
@alexkli just confirming, `---disable-concurrency` is enabled in `wskdebug@1.3.0` correct? (looking at the source, it seems it is: https://github.com/apache/openwhisk-wskdebug/blob/1.3.0/src/agentmgr.js#L188)

**Comment by alexkli** [comment]
@shazron yes.

**Comment by shazron** [comment]
Added the `--disable-concurrency` flag for the Project Firefly app plugin (5.4.0) VS Code launch config: https://github.com/adobe/generator-aio-app/pull/128
